webpack.config.js

```diff
@@ -1,67 +1,70 @@
-const StyleLintPlugin = require('stylelint-webpack-plugin');
-const packageJSON = require('./package.json');
-const path = require('path');
+const StyleLintPlugin = require("stylelint-webpack-plugin");
+const packageJSON = require("./package.json");
+const path = require("path");
+const ESLintPlugin = require("eslint-webpack-plugin");
+const TerserPlugin = require("terser-webpack-plugin");
 
 const DIST = path.resolve("./dist");
-const MODE = process.env.NODE_ENV || 'development';
-const SOURCE_MAP = 'sourec-map';
-const DEVTOOL = (process.env.NODE_ENV === 'development') ? SOURCE_MAP : false;
+const MODE = process.env.NODE_ENV || "development";
+const SOURCE_MAP = "source-map";
+const DEVTOOL = process.env.NODE_ENV === "development" ? SOURCE_MAP : false;
 
-console.log('Webpack mode:', MODE); // eslint-disable-line no-console
+console.log("Webpack mode:", MODE); // eslint-disable-line no-console
+
+const eslintPluginOptions = {
+  // enforce: "pre",
+  // test: /\.js?$/,
```

Comment:
I don't know why I get conflict with these two options, seems `eslint-webpack-plugin` doesn't recognize them as the options...